core/linker/default.h

```diff
@@ -60,7 +64,11 @@ ENTRY(main_entry)
  *       stack from the rest of the protected memories. For this reason the
  *       symbol is arbitrarily small and cannot be defined by the
  *       platform-specific configurations. */
+#if defined (__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U)
 #define STACK_GUARD_BAND 32
+#else
+#define STACK_GUARD_BAND 512
+#endif
```

Comment:
This means the stack guard band is now 512 on Kinetis. Probably not a change you want to make yet.